### ISSUE TITLE
Fix the space between the Up Next episode count

### ISSUE THREAD

**Issue by geekygecko** [Automattic/pocket-casts-android#45]
iOS
<img  width="300" src="https://user-images.githubusercontent.com/308331/177487994-0e0fd16c-d59e-4adb-b03c-1ffd52ea05d2.png" />

Android
<img  width="300" src="https://user-images.githubusercontent.com/308331/177488012-f82f6aba-90e6-453f-90ac-5e995fb652e9.png" />

<img  width="300" src="https://user-images.githubusercontent.com/308331/177488060-fab42ba5-b0f1-4a01-bc0e-126369587283.png" />

**Comment by mchowning** [comment]
Are you talking about how the numbers are kind of smashed together here @geekygecko ?

**Comment by geekygecko** [comment]
Yes, they seem to be much closer than on iOS. Feel free to close this issue if you don't think it's an issue.

<img  width="300" src="https://github.com/Automattic/pocket-casts-android/assets/308331/af668938-b18e-4868-b6d9-000dbdf7c9f2" />

**Comment by mebarbosa** [comment]
Closing this issue since it was fixed after and mini player redesign
